From 7e15b04b20e9149e7f732c2b56c61d3c386d7972 Mon Sep 17 00:00:00 2001
From: Eugene Syromyatnikov <evgsyr@gmail.com>
Date: Mon, 15 Oct 2018 17:47:01 +0200
Subject: [PATCH 171/293] xlat: update siginfo_codes

* xlat/siginfo_codes.in (SI_LWP): Remove, as it is not present on Linux.
(SI_NOINFO): Define only on SPARC, provide fallback value.
---
 xlat/siginfo_codes.in | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/xlat/siginfo_codes.in b/xlat/siginfo_codes.in
index 968c660..b3e0888 100644
--- a/xlat/siginfo_codes.in
+++ b/xlat/siginfo_codes.in
@@ -16,5 +16,7 @@ SI_SIGIO -5
 SI_TKILL -6
 SI_DETHREAD -7
 SI_ASYNCNL -60
-SI_NOINFO
-SI_LWP
+
+#ifdef __sparc__
+SI_NOINFO 32767
+#endif
-- 
1.7.12.4

